DSharpPlus/Clients/DiscordClient.cs

```diff
@@ -864,30 +864,24 @@ private async IAsyncEnumerable<DiscordGuild> GetGuildsInternalAsync
     public async Task<byte[]> HandleHttpInteractionAsync(ArraySegment<byte> body, CancellationToken cancellationToken = default)
     {
         string bodyString = Encoding.UTF8.GetString(body);
-
         JObject data = JObject.Parse(bodyString);
-        
-        DiscordHttpInteraction? interaction = data.ToDiscordObject<DiscordHttpInteraction>();
-        
-        if (interaction is null)
-        {
-            throw new ArgumentException("Unable to parse provided request body to DiscordHttpInteraction");
-        }
-        
+        DiscordHttpInteraction? interaction = data.ToDiscordObject<DiscordHttpInteraction>()
+            ?? throw new ArgumentException("Unable to parse provided request body to DiscordHttpInteraction");
+

```

Comment:
However, for this specific instance I am fairly sure you can remove the nullable notation off of the DiscordHttpInteraction type, as it will throw if it's null anyway.